src/uu/uniq/src/uniq.rs

```diff
@@ -590,13 +599,13 @@ pub fn uumain(args: impl uucore::Args) -> UResult<()> {
 }
 
 pub fn uu_app() -> Command {
-    Command::new(uucore::util_name())
+    let cmd = Command::new(uucore::util_name())
         .version(uucore::crate_version!())
-        .help_template(uucore::localized_help_template(uucore::util_name()))
         .about(translate!("uniq-about"))
         .override_usage(format_usage(&translate!("uniq-usage")))
         .infer_long_args(true)
-        .after_help(translate!("uniq-after-help"))
+        .after_help(translate!("uniq-after-help"));
+    uucore::clap_localization::configure_localized_command(cmd)
```

Comment:
What's the reason for splitting the setup of command into two parts? Why not doing the localization at the end?